docs/getting-started-guides/locally.md

```diff
@@ -17,55 +24,138 @@ At least [Docker](https://docs.docker.com/installation/#installation)
 ps`).  Some of the Kubernetes components need to run as root, which normally
 works fine with docker.
 
-#### etcd
-
-You need an [etcd](https://github.com/coreos/etcd/releases) in your path, please make sure it is installed and in your ``$PATH``.
-
-#### go
-
-You need [go](https://golang.org/doc/install) at least 1.3+ in your path, please make sure it is installed and in your ``$PATH``.
-
 ### Starting the cluster
 
-First, you need to [download Kubernetes](/docs/getting-started-guides/binary_release/). Then open a separate tab of your terminal 
-and run the following (since one needs sudo access to start/stop Kubernetes daemons, it is easier to run the entire script as root):
+Open your terminal and run the following commands:
 
 ```shell
-cd kubernetes
-hack/local-up-cluster.sh
+# Request root access when the prompt is visible and between log lines
+sudo echo "Granting temporary root access"
+
+# Configure versions for the dependencies
+export FLANNEL_VERSION="0.5.5"
+export ETCD_VERSION=2.3.3
+export KUBE_VERSION="1.2.3"
+export KUBERNETES_PROVIDER=local
+export NUM_NODES=1
+
+# You need to replace this with what your IP address is
+export KUBE_API_ADDR=192.168.1.111
+
+# If you haven't downloaded Kubernetes yet, you can do so now
+cd ~
+
+echo "Fetching Kubernetes"
+#curl -L https://github.com/kubernetes/kubernetes/releases/download/v${KUBE_VERSION}/kubernetes.tar.gz \
+#    -o ~/kubernetes.tar.gz
+cp ~/Downloads/kubernetes.tar.gz ~/kubernetes.tar.gz
+tar -zxvf kubernetes.tar.gz
+rm kubernetes.tar.gz
+
+# Extract the version compatible with the current OS
+cd kubernetes/server
+tar -zxvf kubernetes-server-linux-amd64.tar.gz
+
+# Use the binaries we need
```

Comment:
Please, add the optional possibility to move the binaries to `/usr/local/bin/` or add the `~/kube-bin` to `PATH`.